components/Footer.vue

```diff
@@ -23,41 +23,69 @@
 
 <script>
   import {
-    GithubIcon,
     FacebookIcon,
-    TwitterIcon,
+    GithubIcon,
+    GitlabIcon,
+    GlobeIcon,
     InstagramIcon,
-    LinkedinIcon
+    LinkedinIcon,
+    MailIcon,
+    MessageSquareIcon,
+    PhoneIcon,
+    TwitterIcon
   } from 'vue-feather-icons'
-  
+
   export default {
     components: {
-      GithubIcon,
       FacebookIcon,
-      TwitterIcon,
+      GithubIcon,
+      GitlabIcon,
+      GlobeIcon,
       InstagramIcon,
-      LinkedinIcon
+      LinkedinIcon,
+      MailIcon,
+      MessageSquareIcon,
+      PhoneIcon,
+      TwitterIcon
     },
-    
+
     methods: {
       getIconComponentName(contactType) {
         switch (contactType) {
-          case 'github':
-            return 'GithubIcon'
+          case 'email':
+            return 'MailIcon'
           case 'facebook':
             return 'FacebookIcon'
-          case 'twitter':
-            return 'TwitterIcon'
+          case 'github':
+            return 'GithubIcon'
+          case 'gitlab':
+            return 'GitlabIcon'
+          case 'globe':
+            return 'GlobeIcon'
           case 'instagram':
             return 'InstagramIcon'
+          case 'international':
+            return 'GlobeIcon'
           case 'linkedin':
             return 'LinkedinIcon'
+          case 'mail':
+            return 'MailIcon'
+          case 'message':
+            return 'MessageSquareIcon'
+          case 'messenger':
+            return 'MessageSquareIcon'
+          case 'phone':
+            return 'PhoneIcon'
+          case 'twitter':
+            return 'TwitterIcon'
+          case 'web':
+            return 'GlobeIcon'
```

Comment:
I don't think it's a good idea to use a same icon in several cases. You can't really use them at the same time since it'll confuse your users. 

IMHO, we don't need `message `, `email`, `globe` and `international `.

Reply:
That makes sense.  I felt conflicted between using the original name of the icon vs something else that might sound better in context so I just did both.

I amended the relevant commit to remove the ones you mentioned.